jwk.go

```diff
@@ -237,8 +237,21 @@ func fromEcPublicKey(pub *ecdsa.PublicKey) (*rawJsonWebKey, error) {
 }
 
 func (key rawJsonWebKey) rsaPrivateKey() (*rsa.PrivateKey, error) {
-	if key.N == nil || key.E == nil || key.D == nil || key.P == nil || key.Q == nil {
-		return nil, fmt.Errorf("square/go-jose: invalid RSA private key, missing values")
+	var missing byte
```

Comment:
If you make this a `[]string`, you can append and print all missing field values at once (with `strings.Join()`)